static/app/views/performance/transactionSummary/transactionSpans/spanSummary/spanSummaryTable.tsx

```diff
@@ -0,0 +1,280 @@
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
+import {SpanDurationBar} from 'sentry/views/performance/transactionSummary/transactionSpans/spanDetails/spanDetailsTable';
+import {useSpanSummarySort} from 'sentry/views/performance/transactionSummary/transactionSpans/spanSummary/useSpanSummarySort';
+import {renderHeadCell} from 'sentry/views/starfish/components/tableCells/renderHeadCell';
+import {SpanIdCell} from 'sentry/views/starfish/components/tableCells/spanIdCell';
+import {useIndexedSpans} from 'sentry/views/starfish/queries/useIndexedSpans';
+import {
+  type IndexedResponse,
+  SpanIndexedField,
+  type SpanMetricsQueryFilters,
+} from 'sentry/views/starfish/types';
+import {QueryParameterNames} from 'sentry/views/starfish/views/queryParameters';
+
+type DataRowKeys =
+  | SpanIndexedField.ID
+  | SpanIndexedField.TIMESTAMP
+  | SpanIndexedField.SPAN_DURATION
+  | SpanIndexedField.TRANSACTION_ID
+  | SpanIndexedField.TRACE
+  | SpanIndexedField.PROJECT;
+
+type ColumnKeys =
+  | SpanIndexedField.ID
+  | SpanIndexedField.TIMESTAMP
+  | SpanIndexedField.SPAN_DURATION;
+
+type DataRow = Pick<IndexedResponse, DataRowKeys> & {'transaction.duration': number};
+
+type Column = GridColumnHeader<ColumnKeys>;
+
+const COLUMN_ORDER: Column[] = [
+  {
+    key: SpanIndexedField.ID,
+    name: t('Span ID'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: SpanIndexedField.TIMESTAMP,
+    name: t('Timestamp'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+  {
+    key: SpanIndexedField.SPAN_DURATION,
+    name: t('Span Duration'),
+    width: COL_WIDTH_UNDEFINED,
+  },
+];
+
+const COLUMN_TYPE: Omit<
+  Record<ColumnKeys, ColumnType>,
+  'spans' | 'transactionDuration'
+> = {
+  span_id: 'string',
+  timestamp: 'date',
+  'span.duration': 'duration',
+};
+
+const LIMIT = 8;
+
+type Props = {
+  project: Project | undefined;
+};
+
+export default function SpanSummaryTable(props: Props) {
+  const {project} = props;
+  const organization = useOrganization();
+  const {spanSlug} = useParams();
+  const [spanOp, groupId] = spanSlug.split(':');
+
+  const location = useLocation();
+  const {transaction} = location.query;
+  const spansCursor = decodeScalar(location.query?.[QueryParameterNames.SPANS_CURSOR]);
+
+  const filters: SpanMetricsQueryFilters = {
+    'span.group': groupId,
+    'span.op': spanOp,
+    transaction: transaction as string,
+  };
+
+  const sort = useSpanSummarySort();
+
+  const {
+    data: rowData,
+    pageLinks,
+    isLoading: isRowDataLoading,
+  } = useIndexedSpans({
+    fields: [
+      SpanIndexedField.ID,
+      SpanIndexedField.TRANSACTION_ID,
+      SpanIndexedField.TIMESTAMP,
+      SpanIndexedField.SPAN_DURATION,
+      SpanIndexedField.TRACE,
+    ],
+    search: MutableSearch.fromQueryObject(filters),
+    limit: LIMIT,
+    referrer: 'api.performance.span-summary-table',
```

Comment:
you'll have to add all your referrers to the list of allowed referrers in referrer.py and to ALLOWED_EVENTS_REFERRERS (if they aren't already there)

Reply:
I'll open a new PR to add these referrers

Reply:
https://github.com/getsentry/sentry/pull/70466